start.sh

```diff
@@ -0,0 +1,18 @@
+#!bin/bash
+cd ..;
+cd mongod;
+gnome-terminal -e "sudo mongod -dbpath .";
```

Comment:
This would only work in environments where gnome is used. A more generalized method is preferred

Reply:
@iammosespaulr please add changes.

Reply:
on it ... I'll submit a new pr ... and close this in the process